### PR TITLE
Reuse org-wide linter

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: contracts/alphabet/manifest.json [hunk 1]
```diff
@@ -1 +1 @@
...
..."returntype":"Void","safe":false},{"name":"emit","offset":2827,"parameters":[],"returntype":"Void","safe":false},{"name":"gas","offset":2648,"parameters":[],"returntype":"Integer","safe":true},{"name":"name","offset":3458,"parameters":[],"returntype":"String","safe":true},{"name":"neo","offset":2662,"parameters":[],"returntype":"Integer","safe":true},{"name":"onNEP17Payment","offset":1160,"parameters":[{"name":"from","type":"Hash160"},{"name":"amount","type":"Integer"},{"name":"data","type":"Any"}],"returntype":"Void","safe":false},{"name":"update","offset":2516,"parameters":[{"name":"script","type":"ByteArray"},{"name":"manifest","type":"ByteArray"},{"name":"data","type":"Any"}],"returntype":"Void","safe":false},{"name":"verify","offset":3478,"parameters":[],"returntype":"Boolean","safe":true},{"name":"version","offset":3474,"parameters":[],"returntype":"Integer","safe":true},{"name":"vote","offset":3286,"parameters":[{"name":"epoch","type":"Integer"},{"name":"candidates","type":"Array"}],"returntype":"Void","safe":false}],"events":[]},"features":{},"groups":[],"permissions":[{"contract":"*","methods":["update","transfer","vote"]}],"supportedstandards":[],"trusts":[],"extra":null}
```
Comment (carpawell) [inline contracts/alphabet/manifest.json:1]: what changed and why? (have not diffed)

**Review by carpawell — APPROVED** [review]

File: contracts/alphabet/manifest.json (hunk 1 above)
Comment (roman-khimov) [reply contracts/alphabet/manifest.json:1]: range loops in contracts generate a bit different code. Supposedly it'd be a bit more optimal.

Comment (carpawell) [reply contracts/alphabet/manifest.json:1]: `"offset":1160`, i see

Comment (roman-khimov) [reply contracts/alphabet/manifest.json:1]: All contracts got a bit smaller, btw.

**Pull request merged**